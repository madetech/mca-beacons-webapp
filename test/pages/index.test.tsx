import React from "react";
import { render } from "@testing-library/react";
import Home from "../../src/pages/index";

describe("Home Page", () => {
  it.skip("renders correctly", () => {
    const { asFragment } = render(<Home />, {});

    expect(asFragment()).toMatchSnapshot();
  });
});
