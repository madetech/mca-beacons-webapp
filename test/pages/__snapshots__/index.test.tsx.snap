// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page renders correctly 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <div
      class="govuk-accordion"
      data-module="govuk-accordion"
      id="accordion-default"
    >
      <div
        class="govuk-accordion__section "
      >
        <div
          class="govuk-accordion__section-header"
        >
          <h2
            class="govuk-accordion__section-heading"
          >
            <span
              class="govuk-accordion__section-button"
              id="accordion-default-heading-1"
            >
              Writing well for the web
            </span>
          </h2>
        </div>
        <div
          aria-labelledby="accordion-default-heading-1"
          class="govuk-accordion__section-content"
          id="accordion-default-content-1"
        >
          <p
            class="govuk-body"
          >
            This is the content for Writing well for the web.
          </p>
        </div>
      </div>
      <div
        class="govuk-accordion__section "
      >
        <div
          class="govuk-accordion__section-header"
        >
          <h2
            class="govuk-accordion__section-heading"
          >
            <span
              class="govuk-accordion__section-button"
              id="accordion-default-heading-2"
            >
              Writing well for specialists
            </span>
          </h2>
        </div>
        <div
          aria-labelledby="accordion-default-heading-2"
          class="govuk-accordion__section-content"
          id="accordion-default-content-2"
        >
          <p
            class="govuk-body"
          >
            This is the content for Writing well for specialists.
          </p>
        </div>
      </div>
      <div
        class="govuk-accordion__section "
      >
        <div
          class="govuk-accordion__section-header"
        >
          <h2
            class="govuk-accordion__section-heading"
          >
            <span
              class="govuk-accordion__section-button"
              id="accordion-default-heading-3"
            >
              Know your audience
            </span>
          </h2>
        </div>
        <div
          aria-labelledby="accordion-default-heading-3"
          class="govuk-accordion__section-content"
          id="accordion-default-content-3"
        >
          <p
            class="govuk-body"
          >
            This is the content for Know your audience.
          </p>
        </div>
      </div>
      <div
        class="govuk-accordion__section "
      >
        <div
          class="govuk-accordion__section-header"
        >
          <h2
            class="govuk-accordion__section-heading"
          >
            <span
              class="govuk-accordion__section-button"
              id="accordion-default-heading-4"
            >
              How people read
            </span>
          </h2>
        </div>
        <div
          aria-labelledby="accordion-default-heading-4"
          class="govuk-accordion__section-content"
          id="accordion-default-content-4"
        >
          <p
            class="govuk-body"
          >
            This is the content for How people read.
          </p>
        </div>
      </div>
    </div>
    <main
      class="main"
    >
      <h1
        class="title"
      >
        Welcome to 
        <a
          href="https://nextjs.org"
        >
          Next.js!
        </a>
      </h1>
      <p
        class="description"
      >
        Get started by editing 
        <code
          class="code"
        >
          pages/index.js
        </code>
      </p>
      <div
        class="grid"
      >
        <a
          class="card"
          href="https://nextjs.org/docs"
        >
          <h3>
            Documentation →
          </h3>
          <p>
            Find in-depth information about Next.js features and API.
          </p>
        </a>
        <a
          class="card"
          href="https://nextjs.org/learn"
        >
          <h3>
            Learn →
          </h3>
          <p>
            Learn about Next.js in an interactive course with quizzes!
          </p>
        </a>
        <a
          class="card"
          href="https://github.com/vercel/next.js/tree/master/examples"
        >
          <h3>
            Examples →
          </h3>
          <p>
            Discover and deploy boilerplate example Next.js projects.
          </p>
        </a>
        <a
          class="card"
          href="https://vercel.com/import?filter=next.js&utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
        >
          <h3>
            Deploy →
          </h3>
          <p>
            Instantly deploy your Next.js site to a public URL with Vercel.
          </p>
        </a>
      </div>
    </main>
    <footer
      class="footer"
    >
      <a
        href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
        rel="noopener noreferrer"
        target="_blank"
      >
        Powered by 
        <img
          alt="Vercel Logo"
          class="logo"
          src="/vercel.svg"
        />
      </a>
    </footer>
    <div
      class="govuk-tabs"
      data-module="govuk-tabs"
    >
      <h2
        class="govuk-tabs__title"
      >
        Contents
      </h2>
      <ul
        class="govuk-tabs__list"
      >
        <li
          class="govuk-tabs__list-item govuk-tabs__list-item--selected"
        >
          <a
            class="govuk-tabs__tab"
            href="#past-day"
          >
            Past day
          </a>
        </li>
        <li
          class="govuk-tabs__list-item"
        >
          <a
            class="govuk-tabs__tab"
            href="#past-week"
          >
            Past week
          </a>
        </li>
        <li
          class="govuk-tabs__list-item"
        >
          <a
            class="govuk-tabs__tab"
            href="#past-month"
          >
            Past month
          </a>
        </li>
        <li
          class="govuk-tabs__list-item"
        >
          <a
            class="govuk-tabs__tab"
            href="#past-year"
          >
            Past year
          </a>
        </li>
      </ul>
      <div
        class="govuk-tabs__panel"
        id="past-day"
      >
        <h2
          class="govuk-heading-l"
        >
          Past day
        </h2>
        <table
          class="govuk-table"
        >
          <thead
            class="govuk-table__head"
          >
            <tr
              class="govuk-table__row"
            >
              <th
                class="govuk-table__header"
                scope="col"
              >
                Case manager
              </th>
              <th
                class="govuk-table__header"
                scope="col"
              >
                Cases opened
              </th>
              <th
                class="govuk-table__header"
                scope="col"
              >
                Cases closed
              </th>
            </tr>
          </thead>
          <tbody
            class="govuk-table__body"
          >
            <tr
              class="govuk-table__row"
            >
              <td
                class="govuk-table__cell"
              >
                David Francis
              </td>
              <td
                class="govuk-table__cell"
              >
                3
              </td>
              <td
                class="govuk-table__cell"
              >
                0
              </td>
            </tr>
            <tr
              class="govuk-table__row"
            >
              <td
                class="govuk-table__cell"
              >
                Paul Farmer
              </td>
              <td
                class="govuk-table__cell"
              >
                1
              </td>
              <td
                class="govuk-table__cell"
              >
                0
              </td>
            </tr>
            <tr
              class="govuk-table__row"
            >
              <td
                class="govuk-table__cell"
              >
                Rita Patel
              </td>
              <td
                class="govuk-table__cell"
              >
                2
              </td>
              <td
                class="govuk-table__cell"
              >
                0
              </td>
            </tr>
          </tbody>
        </table>
      </div>
      <div
        class="govuk-tabs__panel govuk-tabs__panel--hidden"
        id="past-week"
      >
        <h2
          class="govuk-heading-l"
        >
          Past week
        </h2>
        <table
          class="govuk-table"
        >
          <thead
            class="govuk-table__head"
          >
            <tr
              class="govuk-table__row"
            >
              <th
                class="govuk-table__header"
                scope="col"
              >
                Case manager
              </th>
              <th
                class="govuk-table__header"
                scope="col"
              >
                Cases opened
              </th>
              <th
                class="govuk-table__header"
                scope="col"
              >
                Cases closed
              </th>
            </tr>
          </thead>
          <tbody
            class="govuk-table__body"
          >
            <tr
              class="govuk-table__row"
            >
              <td
                class="govuk-table__cell"
              >
                David Francis
              </td>
              <td
                class="govuk-table__cell"
              >
                24
              </td>
              <td
                class="govuk-table__cell"
              >
                18
              </td>
            </tr>
            <tr
              class="govuk-table__row"
            >
              <td
                class="govuk-table__cell"
              >
                Paul Farmer
              </td>
              <td
                class="govuk-table__cell"
              >
                16
              </td>
              <td
                class="govuk-table__cell"
              >
                20
              </td>
            </tr>
            <tr
              class="govuk-table__row"
            >
              <td
                class="govuk-table__cell"
              >
                Rita Patel
              </td>
              <td
                class="govuk-table__cell"
              >
                24
              </td>
              <td
                class="govuk-table__cell"
              >
                27
              </td>
            </tr>
          </tbody>
        </table>
      </div>
      <div
        class="govuk-tabs__panel govuk-tabs__panel--hidden"
        id="past-month"
      >
        <h2
          class="govuk-heading-l"
        >
          Past month
        </h2>
        <table
          class="govuk-table"
        >
          <thead
            class="govuk-table__head"
          >
            <tr
              class="govuk-table__row"
            >
              <th
                class="govuk-table__header"
                scope="col"
              >
                Case manager
              </th>
              <th
                class="govuk-table__header"
                scope="col"
              >
                Cases opened
              </th>
              <th
                class="govuk-table__header"
                scope="col"
              >
                Cases closed
              </th>
            </tr>
          </thead>
          <tbody
            class="govuk-table__body"
          >
            <tr
              class="govuk-table__row"
            >
              <td
                class="govuk-table__cell"
              >
                David Francis
              </td>
              <td
                class="govuk-table__cell"
              >
                98
              </td>
              <td
                class="govuk-table__cell"
              >
                95
              </td>
            </tr>
            <tr
              class="govuk-table__row"
            >
              <td
                class="govuk-table__cell"
              >
                Paul Farmer
              </td>
              <td
                class="govuk-table__cell"
              >
                122
              </td>
              <td
                class="govuk-table__cell"
              >
                131
              </td>
            </tr>
            <tr
              class="govuk-table__row"
            >
              <td
                class="govuk-table__cell"
              >
                Rita Patel
              </td>
              <td
                class="govuk-table__cell"
              >
                126
              </td>
              <td
                class="govuk-table__cell"
              >
                142
              </td>
            </tr>
          </tbody>
        </table>
      </div>
      <div
        class="govuk-tabs__panel govuk-tabs__panel--hidden"
        id="past-year"
      >
        <h2
          class="govuk-heading-l"
        >
          Past year
        </h2>
        <table
          class="govuk-table"
        >
          <thead
            class="govuk-table__head"
          >
            <tr
              class="govuk-table__row"
            >
              <th
                class="govuk-table__header"
                scope="col"
              >
                Case manager
              </th>
              <th
                class="govuk-table__header"
                scope="col"
              >
                Cases opened
              </th>
              <th
                class="govuk-table__header"
                scope="col"
              >
                Cases closed
              </th>
            </tr>
          </thead>
          <tbody
            class="govuk-table__body"
          >
            <tr
              class="govuk-table__row"
            >
              <td
                class="govuk-table__cell"
              >
                David Francis
              </td>
              <td
                class="govuk-table__cell"
              >
                1380
              </td>
              <td
                class="govuk-table__cell"
              >
                1472
              </td>
            </tr>
            <tr
              class="govuk-table__row"
            >
              <td
                class="govuk-table__cell"
              >
                Paul Farmer
              </td>
              <td
                class="govuk-table__cell"
              >
                1129
              </td>
              <td
                class="govuk-table__cell"
              >
                1083
              </td>
            </tr>
            <tr
              class="govuk-table__row"
            >
              <td
                class="govuk-table__cell"
              >
                Rita Patel
              </td>
              <td
                class="govuk-table__cell"
              >
                1539
              </td>
              <td
                class="govuk-table__cell"
              >
                1265
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
